RELEASE-NOTES.txt

```diff
@@ -10,6 +10,7 @@
 * [*] Block Editor: Add GIF badge for animated GIFs uploaded to Image blocks [https://github.com/WordPress/gutenberg/pull/38996]
 * [*] Block Editor: Small refinement to media upload errors, including centering and tweaking copy. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4597]
 * [*] Block Editor: Fix issue with list's starting index and the order [https://github.com/WordPress/gutenberg/pull/39354]
+* [*] My Site: Fixes an issue where a deleted site is displayed until the app restarts [https://github.com/wordpress-mobile/WordPress-Android/pull/16063]
```

Comment:
@SowmiaS Hi 👋🏼 , Could you move the line describing your fix to the milestone - 19.6. Am sorry for not communicating it earlier or updating the milestone tag in PR.

Reply:
No problem at all @AjeshRPai 🙂. I have updated the RELEASE-NOTES.txt, moving the release note related to this PR from milestone 19.5 to 19.6. Done in 6b5f5e96d95bbe11ee36dd54da23c30e451ef067